config/initializers/okcomputer.rb

```diff
@@ -6,12 +6,37 @@
 # Required
 OkComputer::Registry.register "ruby_version", OkComputer::RubyVersionCheck.new
 OkComputer::Registry.register "redis", OkComputer::RedisCheck.new(url: ENV.fetch("REDIS_URL", Settings.redis_url))
-if Settings.rabbitmq.enabled
-  OkComputer::Registry.register "rabbit",
-    OkComputer::RabbitmqCheck.new(hostname: Settings.rabbitmq.hostname,
+
+class RabbitQueueExistsCheck < OkComputer::Check
+  attr_accessor :queue_names, :conn
+
+  def initialize(queue_names)
+    self.queue_names = Array(queue_names)
+    self.conn = Bunny.new(hostname: Settings.rabbitmq.hostname,
```

Comment:
I was a bit surprised to see `self.` here instead of `@`. I don't think there's a functional difference in this case.